src/components/Docs/SnippetViewer/WriteRequestViewer.tsx

```diff
@@ -23,8 +18,32 @@ interface WriteRequestViewerOpts {
   allowedLanguages?: SupportedLanguage[];
 }
 
-function writeRequestViewer(lang: SupportedLanguage, opts: WriteRequestViewerOpts, languageMappings: LanguageMappings) {
+function writeRequestViewer(lang: SupportedLanguage, opts: WriteRequestViewerOpts) {
+  const modelId = opts.authorizationModelId ? opts.authorizationModelId : DefaultAuthorizationModelId;
   switch (lang) {
+    case SupportedLanguage.CLI: {
+      return `${
+        opts.relationshipTuples?.length
+          ? opts.relationshipTuples
+              .map(
+                (tuple) =>
+                  `fga write --store-id=\${FGA_STORE_ID} --model-id=${modelId} ${tuple.user} ${tuple.relation} ${tuple.object}`,
```

Comment:
can you verify whether this will work if you have multiple tuples?  Will it result in multiple fga write line instead of a single fga write with multiple tuples?

Reply:
Yes, the cli does not support multiple requests in a single write unless you are doing an import